src/extensions/ERC20.sol

```diff
@@ -10,30 +10,28 @@ interface ERC20 {
     function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
```

Comment:
Should rename this file and interface to `IGogeERC20.sol` and `IGogeERC20`. The interface is exclusive to `GogeToken.sol`, not regular ERC20, and this file defines an interface not a contract.

Reply:
That's a good idea. Will do this